resources/main.js

```diff
@@ -4,4 +4,11 @@ $(document).ready(function () {
         edge: 'right',
         closeOnClick: true
     });
+
+    $('#analytics-link').on('click', function() {
```

Comment:
Hahaha looked over that one, still moving this to a separate function isn't a bad idea.